Assets/MixedRealityToolkit.Tests/TestUtilities.cs

```diff
@@ -22,23 +21,83 @@ namespace Microsoft.MixedReality.Toolkit.Tests
 {
     public static class TestUtilities
     {
-        public static Scene testScene;
+        const string primaryTestSceneTemporarySavePath = "Assets/__temp_primary_test_scene.unity";
+        const string additiveTestSceneTemporarySavePath = "Assets/__temp_additive_test_scene_#.unity";
+        public static Scene primaryTestScene;
+        public static Scene[] additiveTestScenes = new Scene[0];
 
         public static void InitializeMixedRealityToolkit()
         {
             MixedRealityToolkit.ConfirmInitialized();
         }
 
-        public static void CleanupScene()
+        /// <summary>
+        /// Destroyes all scene assets that were created over the course of testing
+        /// </summary>
+        public static void TearDownScenes()
         {
-            // Create a default test scene.
+#if UNITY_EDITOR
+            // If any of our scenes were saved, tear down the assets
+            SceneAsset primaryTestSceneAsset = AssetDatabase.LoadAssetAtPath<SceneAsset>(primaryTestSceneTemporarySavePath);
+            if (primaryTestSceneAsset != null)
+            {
+                AssetDatabase.DeleteAsset(primaryTestSceneTemporarySavePath);
+            }
+
+            for (int i = 0; i < additiveTestScenes.Length; i++)
+            {
+                string path = additiveTestSceneTemporarySavePath.Replace("#", i.ToString());
+                SceneAsset additiveTestSceneAsset = AssetDatabase.LoadAssetAtPath<SceneAsset>(path);
+                if (additiveTestSceneAsset != null)
+                {
+                    AssetDatabase.DeleteAsset(path);
+                }
+            }
+            AssetDatabase.Refresh();
+#endif
+        }
+
+        /// <summary>
+        /// Creates a number of scenes and loads them additively for testing. Must create a minimum of 1.
```

Comment:
Which scene becomes the active scene, after this is loaded? Is it the last one? Should we make it configurable.

Reply:
The latest loaded is always the active scene unless you set it manually.